samples/cellular/modem_shell/src/utils/net_utils.c

```diff
@@ -17,13 +17,10 @@
 
 int net_utils_socket_pdn_id_set(int fd, uint32_t pdn_id)
 {
-	int ret;
+	int err = setsockopt(fd, SOL_SOCKET, SO_BINDTOPDN, &pdn_id, sizeof(pdn_id));
 
-	ret = setsockopt(fd, SOL_SOCKET, SO_BINDTOPDN, &pdn_id, sizeof(pdn_id));
-	if (ret < 0) {
-		mosh_error(
-			"Failed to bind socket with PDN ID %d, error: %d, %s",
-			pdn_id, ret, strerror(ret));
```

Comment:
Was there no value from `strerror` in the case you've seen?

Reply:
I want to align this error message with other error messages, and no-one else uses `strerror()`.

Reply:
That's fine. I was just curious what `strerror` was showing.

Reply:
`strerror(-1)` show nothing.

Reply:
maybe it should have been strerror(errno) originally